routes/main.go

```diff
@@ -0,0 +1,15 @@
+package routes
+
+import "github.com/gin-gonic/gin"
```

Comment:
why are we using gin and not mux ?

Reply:
- Yes, any reason to pick mux over gin?

Reply:
@yesyash and @bhtibrewal  flexibility or performance ...?

Reply:
- There are a few reasons to pick Gin over mux @bhtibrewal :
    1. Recently the Gorilla toolkit was going to be archived and then went under transfer for the maintainers thus I am a bit uncertain about how they want to take this toolkit forward.
    2. I have seen GIN perform better than MUX.